controllers/admin/AdminAddressesController.php

```diff
@@ -444,12 +444,6 @@ protected function processAddressFormat()
 
         $selected_country = ($tmp_addr && $tmp_addr->id_country) ? $tmp_addr->id_country : (int)Configuration::get('PS_COUNTRY_DEFAULT');
 
-        $inv_adr_fields = AddressFormat::getOrderedAddressFields($selected_country, false, true);
-        $dlv_adr_fields = AddressFormat::getOrderedAddressFields($selected_country, false, true);
-
-        $inv_all_fields = array();
-        $dlv_all_fields = array();
```

Comment:
IMHO this method is unclear. Maybe efficient and concise but really difficult to read and understand. This led @eternoendless to remove actually used vars.
Maybe this is a good time to refacto this method.